packages/victory-vendor/package.json

```diff
@@ -16,10 +16,13 @@
   "author": "Formidable",
   "license": "MIT AND ISC",
   "exports": {
+    "./package.json": "./package.json",
     "./d3-*": {
       "import": "./es/d3-*.js",
       "default": "./lib/d3-*.js"
-    }
+    },
+    "./es/d3-*": "./es/d3-*.js",
+    "./lib/d3-*": "./lib/d3-*.js"
```

Comment:
Another random question ...
Do Victory consumers import `victory-vendor` directly?  I was under the impression that this was just an internal module.
If it's just internal, then we probably don't need to have explicit `es` and `lib` export paths, since internally we don't use them, so these exports would only be available externally, and might result in 2 copies of the dependency being used.  Does that make sense?

Reply:
I don't expect external consumers to likely need this -- I was more thinking of a way to be able to force the right library to use if the runtime choose something different, as I hit some issues with Jest being in one Mode when the execution environment was different.

But as we don't have that problem now, I can just remove the entries.